sql2.txt

```diff
@@ -17,36 +28,71 @@ may have ordered the same product in several different orders.  So the total qua
 SUM(Quantity) AS TotalQuantity and you must GROUP BY CustomerID, ProductID.  Paste your
 SQL statement below.
 
-
+SELECT CustomerName, ProductName, SUM(Quantity) AS TotalQuantity FROM Customers c
+JOIN Orders o
+ON c.CustomerID = o.CustomerID
+JOIN OrderDetails od
+On o.OrderID = od.OrderID
+JOIN Products p
+on od.ProductID = p.ProductID
+GROUP BY c.CustomerID, p.ProductID
+ORDER BY CustomerName ASC;
 
 4. Extend the previous query to give the total price as well as the customer name, product name, 
 and total quantity.  Here you will have
 (SUM(Quantity) * Price) AS TotalPrice.  Paste your SQL statement below.
 
+SELECT CustomerName, ProductName, (SUM(Quantity) * Price) AS TotalPrice FROM Customers c
+JOIN Orders o
+ON c.CustomerID = o.CustomerID
+JOIN OrderDetails od
+On o.OrderID = od.OrderID
+JOIN Products p
+on od.ProductID = p.ProductID
+GROUP BY c.CustomerID, p.ProductID
+ORDER BY CustomerName ASC;
 
 5. Add a new Order to the Orders table.  You will have to specify the ID of an existing customer
 to satisfy the foreign key constraint.  You will also have to specify the ID of an existing
 employee.  Paste your SQL statement below.
 
+INSERT INTO Orders
+(CustomerID, EmployeeID, ShipperID)
+VALUES (90, 5, 3);
 
 6. Add three OrderDetails.  These should specify the ID of the new Order you just created.  For each,
 you will have to specify the ID of an existing product.  Paste your SQL statement below.
 
+INSERT INTO OrderDetails
+(OrderID, ProductID, Quantity)
+VALUES (10248, 23, 1),
+(10248, 24, 1),
+(10248, 25, 1);
 
 7.  Delete the third of the OrderDetails you just added.  Paste your SQL statement below.
 
-
+DELETE FROM OrderDetails
+WHERE OrderID = 10248
+AND ProductID = 25;
 
 8. Update the Products table, so that every product with a price less than $20 has the price increased by
 0.50.  Paste your SQL statement below.
 
+UPDATE Products
+SET Price = (Price * 0.50)
```

Comment:
The requirement is:

> the price increased by 0.50

but this does something else. Could you tell me what that is, and then fix it?

Reply:
Oh man I halved the prices 😅 I'll fix it